helpers/admin/adminHelpers.ts

```diff
@@ -79,7 +79,7 @@ export const errorCheckSingleField = async (
   // title is the name of field being checked
   const { title } = properties[propertyIndex]
   try {
-    await reach(schema, title).validate(data[title])
+    await reach(schema, title, null, null).validate(data[title])
```

Comment:
Had to change this because `yup` broke the type-check with this error:

```
helpers/admin/adminHelpers.ts:82:11 - error TS2554: Expected 4 arguments, but got 2.

82     await reach(schema, title).validate(data[title])
             ~~~~~~~~~~~~~~~~~~~~

  node_modules/yup/lib/util/reach.d.ts:6:46
    6 declare const reach: (obj: {}, path: string, value: any, context: any) => any;
                                                   ~~~~~~~~~~
    An argument for 'value' was not provided.
```

The `yup` API docs says the last 2 arguments are optional, but the type def says it's required.